packages/webview_flutter/webview_flutter_web/example/pubspec.yaml

```diff
@@ -10,7 +10,8 @@ dependencies:
     sdk: flutter
   flutter_web_plugins:
     sdk: flutter
-  webview_flutter_platform_interface: ^1.8.0
+  webview_flutter_platform_interface:
+    path: ../../webview_flutter_platform_interface
```

Comment:
Should we have a warning comment on this to remind us not to land it this way in the main branch? Maybe even do a dependency_override like we do for multi-package PRs?

Reply:
Ah, I forgot about the tool command that does this. I updated the pubspecs.